jwst/datamodels/schemas/msatargacq.schema.yaml

```diff
@@ -0,0 +1,201 @@
+%YAML 1.1
```

Comment:
Why are there 2 separate schemas files (`msa_targ_acq.schema` above and now this one here) that both define `msa_targ_acq_table` and its contents? Only need one of these. This is defining the same table in 2 different schemas. See the group.schema or int_times.schema for examples of schemas for table extensions that are included within another higher-level product schema.

Reply:
I am deleting msa_targ_acq.schema.yaml in favor of msatargacq.schema.yaml for consistent naming.